From 764392435231f53631f26ac718ac9a295a446056 Mon Sep 17 00:00:00 2001
From: Ole Streicher <debian@liska.ath.cx>
Date: Wed, 5 Feb 2014 09:15:08 +0900
Subject: Don't install the test script "measuresdata.csh" to /usr/bin

 This script is just a test script, so it should go into the examples.
---
 measures/apps/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/measures/apps/CMakeLists.txt b/measures/apps/CMakeLists.txt
index 1901418..b888c9d 100644
--- a/measures/apps/CMakeLists.txt
+++ b/measures/apps/CMakeLists.txt
@@ -7,4 +7,4 @@ endforeach(prog findmeastable)
 add_executable (measuresdata  measuresdata/measuresdata.cc)
 target_link_libraries (measuresdata casa_measures)
 install(TARGETS measuresdata DESTINATION bin)
-install(PROGRAMS measuresdata/measuresdata.csh DESTINATION bin)
+install(PROGRAMS DESTINATION bin)
